winterfell/webapp/src/js/profileCtrl.js

```diff
@@ -36,7 +15,9 @@ app.controller('profileCtrl', ['$scope', '$location', '$window', 'Profile', 'cla
 
     function init() {
       for (var i = 0; i < $scope.claims.length; i++) {
+        $scope.claims[i].id = $scope.claims[i].externalId; // todo, server shouldn't be sending rowId and externalId. We only reference claims through claim.id
         $scope.claims[i].header = createHeaderString($scope.claims[i]);
+        $scope.claims[i].date = utils.formatServerDate($scope.claims[i].stateUpdatedAt);
```

Comment:
You could use the pattern in `formCtrl.js`, which is to inject `$filter` and then call `$filter('date')(new Date(), 'MM/dd/yyyy');`, this uses angulars built in date formatter